package.json

```diff
@@ -36,121 +36,121 @@
-      "types": "./dist/thing/build.d.ts"
+      "types": "./dist/thing/build.d.ts",
+      "import": "./dist/thing/build.mjs"
     },
     "./thing/mock": {
-      "import": "./dist/thing/mock.mjs",
-      "types": "./dist/thing/mock.d.ts"
+      "types": "./dist/thing/mock.d.ts",
+      "import": "./dist/thing/mock.mjs"
     },
     "./acl/acl": {
-      "import": "./dist/acl/acl.mjs",
-      "types": "./dist/acl/acl.d.ts"
+      "types": "./dist/acl/acl.d.ts",
+      "import": "./dist/acl/acl.mjs"
     },
     "./acl/agent": {
-      "import": "./dist/acl/agent.mjs",
-      "types": "./dist/acl/agent.d.ts"
+      "types": "./dist/acl/agent.d.ts",
+      "import": "./dist/acl/agent.mjs"
     },
     "./acl/group": {
-      "import": "./dist/acl/group.mjs",
-      "types": "./dist/acl/group.d.ts"
+      "types": "./dist/acl/group.d.ts",
+      "import": "./dist/acl/group.mjs"
     },
     "./acl/class": {
-      "import": "./dist/acl/class.mjs",
-      "types": "./dist/acl/class.d.ts"
+      "types": "./dist/acl/class.d.ts",
+      "import": "./dist/acl/class.mjs"
     },
     "./acl/mock": {
-      "import": "./dist/acl/mock.mjs",
-      "types": "./dist/acl/mock.d.ts"
+      "types": "./dist/acl/mock.d.ts",
+      "import": "./dist/acl/mock.mjs"
     },
     "./acp/acp": {
-      "import": "./dist/acp/acp.mjs",
-      "types": "./dist/acp/acp.d.ts"
+      "types": "./dist/acp/acp.d.ts",
+      "import": "./dist/acp/acp.mjs"
     },
     "./acp/control": {
-      "import": "./dist/acp/control.mjs",
-      "types": "./dist/acp/control.d.ts"
+      "types": "./dist/acp/control.d.ts",
+      "import": "./dist/acp/control.mjs"
     },
     "./acp/policy": {
-      "import": "./dist/acp/policy.mjs",
-      "types": "./dist/acp/policy.d.ts"
+      "types": "./dist/acp/policy.d.ts",
+      "import": "./dist/acp/policy.mjs"
     },
     "./acp/rule": {
-      "import": "./dist/acp/rule.mjs",
-      "types": "./dist/acp/rule.d.ts"
+      "types": "./dist/acp/rule.d.ts",
+      "import": "./dist/acp/rule.mjs"
     },
     "./acp/mock": {
-      "import": "./dist/acp/mock.mjs",
-      "types": "./dist/acp/mock.d.ts"
+      "types": "./dist/acp/mock.d.ts",
+      "import": "./dist/acp/mock.mjs"
     },
     "./access/universal": {
-      "import": "./dist/access/universal.mjs",
-      "types": "./dist/access/universal.d.ts"
+      "types": "./dist/access/universal.d.ts",
+      "import": "./dist/access/universal.mjs"
     },
     "./rdfjs": {
-      "import": "./dist/rdfjs.mjs",
-      "types": "./dist/rdfjs.d.ts"
+      "types": "./dist/rdfjs.d.ts",
+      "import": "./dist/rdfjs.mjs"
     },
     "./profile/jwks": {
-      "import": "./dist/profile/jwks.mjs",
-      "types": "./dist/profile/jwks.d.ts"
+      "types": "./dist/profile/jwks.d.ts",
+      "import": "./dist/profile/jwks.mjs"
     },
     "./profile/webId": {
-      "import": "./dist/profile/webId.mjs",
-      "types": "./dist/profile/webId.d.ts"
+      "types": "./dist/profile/webid.d.ts",
```

Comment:
🙌

Reply:
@NSeydoux I've also added in a subpath export for `"./profile/webid"` in lowercase to encourage that usage instead of the camelCase one.